scripts/monitoring/cron-send-project-count.sh

```diff
@@ -0,0 +1,12 @@
+!#/bin/bash
+
+#get the number of projects except which are ours
+projects=$(oca get projects | egrep -Ev 'NAME|default|openshift|openshift-infra' | wc -l)
```

Comment:
Is this working from the subshell?

Can you error out the return code if this command fails?